### DISCUSSION TITLE
Since new update Clients can't connect even after resettlement

### DISCUSSION THREAD

**Discussion by SECRET-GUEST** [RaspAP/raspap-webgui#1222] [General]
**Hi there !**

I think I'm about to know all the troubleshootings documentation but I can follow any steps I want it's not fixing my issue .

**What I have/ I did** ;

- I'm using a **Raspberry 3b.**
- **Problem's not coming from the charge** (I have an official one but **I tried many other 5.1v** and fast charge it didn't solved the problem).
- **I have a good and stable network quality**.
- **Issue appeard right after the new update** yesterday.
- My **location isn't the problem**, I mean the device location.
- I have VPN/ TOR directly on my devices, **not on RPI**.
- **I have reset all installation even from 0** but I couldn't connect to SSID "Webgui" .
|-> So for sure the password isn't the problem (privileges neither, it's obvious but I prefere to write it x)
- I'm using this **RPI only for raspap and nothing else**.

Icing on the cake ; my smartsh*t like **plugs or lamps are still connected to the RPI** and works perfectly when I use the original SD card but for everything else smartphone, tabs, pc impossible to connect.

But to be sure I'm missing nothing  ; 

**After the update I was able to connect only one device** 
=> I had to disconnect my computer to use my phone for example

But now nothing is working exept as I wrote the domotic

**### Thanks for help hope this report will be useful for someone !**

**Comment by billz** [comment]
You can enable logging for both hostapd and dhcp to diagnose these issues.
https://docs.raspap.com/ap-basics/#troubleshooting

Also, if this is an upgrade and you changed any settings, be sure first to restart the hotspot from the UI.

**Reply by SECRET-GUEST** [reply]
As I said I used another SD card to make test after I discovered there was no issues at all found by any systems so I changed nothing and all diagnostics was empty of possible solution that's why I'm posting this here btw.

But for some reason today the sh*t works again after a dist-upgrade 🤷🏻 ( I obviously also tried this before)

I suppose maybe the pi or internet get a surge during the update and a file was lightly corrupted (???)  not enough to be detected but I don't know it's just complot theory at this point ..

**Reply by billz** [reply]
Tough to say but it's possible. We recommend a `dist-upgrade` or `full-upgrade` before installing or upgrading RaspAP, followed by a reboot. https://docs.raspap.com/#quick-start
